bundler/bundler.gemspec

```diff
@@ -24,10 +24,10 @@ Gem::Specification.new do |s|
 
   if s.respond_to?(:metadata=)
     s.metadata = {
-      "bug_tracker_uri" => "https://github.com/rubygems/bundler/issues",
-      "changelog_uri" => "https://github.com/rubygems/bundler/blob/master/CHANGELOG.md",
+      "bug_tracker_uri" => "https://github.com/rubygems/rubygems/issues",
```

Comment:
Is it possible to use query string parameters to point out Issues with the Bundler Label?

Reply:
We could, but similar to https://github.com/rubygems/rubygems/pull/3606#discussion_r421697145, I’m not sure whether this would be better. If you prefer, though, we can update the link to [include a filter](https://github.com/rubygems/rubygems/issues?q=is%3Aopen+is%3Aissue+label%3ABundler).

Reply:
Right, yep, _that_'s better.

Reply:
Should we also change other instances to filter by the “Bundler” label or should we just update this `bug_tracker_uri`?

Reply:
I think the filter is pretty smart for this. So, I'd use it for all "used to be Bundler Issues" places..

Reply:
Changed in 96dc57de0069ab82b47126c8c9729eed1db383fe. Analogously, there are some existing links which should now filter by the “RubyGems” label but I’d rather do that in a separate pull request as to focus on updating [rubygems/bundler](https://github.com/rubygems/bundler) links in this one.